spec/factories/reservation.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :reservation do
+    screening_id {1}
```

Comment:
that won't work unless there is a screening already in the database, and every spec starts with empty db
```suggestion
    screening
```
This will look for a screening factory and create a screening for you